cli/pkg/cmd/flags.go

```diff
@@ -28,11 +28,22 @@ func ValidateUpdateSliceFlags(cmd *cobra.Command, args []string) error {
 const (
 	flagDoRunMigrations = "do-migrations"
 	flagSkipMigrations  = "skip-migrations"
+	flagImageSrcEnv     = "image-src-env"
+	flagImageSrcStack   = "image-src-stack"
 )
 
 func SetMigrationFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("do-migrations", true, "Specify if you want to force migrations to run")
-	cmd.Flags().Bool("skip-migrations", false, "Specify if you want to skip migrations")
+	cmd.Flags().Bool(flagDoRunMigrations, true, "Specify if you want to force migrations to run")
+	cmd.Flags().Bool(flagSkipMigrations, false, "Specify if you want to skip migrations")
+}
+
+func SetImagePromotionFlags(cmd *cobra.Command, imageSrcEnv, imageSrcStack *string) {
+	cmd.Flags().StringVar(imageSrcEnv, flagImageSrcEnv, "", "Will promote an image from a specified environment. Must be used with image-src-stack")
```

Comment:
Can we keep "promote" out of the UI? I don't want to imply that's the only thing this feature could/should be used for. I think "pull" might be a better way to describe this?

Reply:
changed all the "promote" lingo to pull and push lingo, which I think is more generic